code/headers/plant_health_monitor.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <base_module.hpp>
+#include <hwlib.hpp>
+#include <moisture_sensor.hpp>
+
+namespace r2d2::plant_rescue_system {
+
+    /**
+     * Class plant_health_monitor_c is Association from moisture sensor.
+     * diferend sensor can be implemented in this class to get a clear status of
+     * the health of a plant.
+     */
+    class plant_health_monitor_c : public base_module_c {
```

Comment:
This has to be called module_c like all other modules. The name difference is found in the namespace.

Reply:
i looked in the style guide and it said nothing about name module_c

Reply:
I think it was something that came later in the project, but if it is not in there. Then this is technically correct